packages/profile-sync-controller/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.3.0]
+
+### Added
+
+- Add a `customProvider` option to the sdk `JwtBearerAuth` class ([#5105](https://github.com/MetaMask/core/pull/5105))
+
+### Changed
+
+- Bump `eslint` to `^9.11.1` and migrate to flat config ([#4727](https://github.com/MetaMask/core/pull/4727))
+- Bump `@metamask/keyring-api` from `^12.0.0` to `^13.0.0` and `@metamask/keyring-internal-api` from `^1.0.0` to `^1.1.0` ([#5066](https://github.com/MetaMask/core/pull/5066))
```

Comment:
Hmmm, will need to double check docs. Since this includes a "major" dependency bump, should our package also be a major bump?

Reply:
Technically looking at the PR: https://github.com/MetaMask/core/pull/5066
It seems like for this package it was not a breaking change, so we could stick with a minor version bump.

However we can raise if this should be a major bump for consistency.